src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/WebClientConfig.kt

```diff
@@ -34,37 +34,27 @@ class WebClientConfig(
       )
       .build()
```

Comment:
Given we have `AuthenticatedWebClientConfig` maybe rename this class to e.g. `NoAuthWebClientConfig`?